README.md

```diff
@@ -15,8 +15,12 @@ Build the library using Maven 3.3.9 or newer as shown below:
 git clone https://github.com/linkedin/spark-tfrecord.git
 cd spark-tfrecord
 mvn clean install
-# one can specific the spark version and tensorflow hadoop version
-# mvn clean install -Dspark.version=2.2.1 -Dtensorflow.hadoop.version=1.15.0
+
+# One can specify the spark version and tensorflow hadoop version, for example
+mvn clean install -Dspark.version=2.4.6 -Dtensorflow.hadoop.version=1.15.0
+
+# Or for building with Spark 3, use the following
```

Comment:
Could you add suggestion to change `artifactId` to `spark-tfrecord_2.12` in pom.xml  when build with spark 3.x?

Reply:
BTW, this PR is already ready to merge, could you squash the commits?